packages/graphql/src/examples/starwars/schema/store.ts

```diff
@@ -0,0 +1,104 @@
+import { Droid, Episode, Human, StarWarsStore } from './types';
+
+/**
+ * This is an example of a real store to accompany the schema.
+ * We can create a simulacrum store that matches the _shape_ of this one,
+ * and place it in context to run the schema as a simulation
+ */
```

Comment:
This store isn't actually used anywhere in-code (we only wire-up / test the simulated store). It does seem to have some value, if only to illustrate the workflow of having a simulated store that matches the shape of the real one.

Any thoughts on whether we should remove this, or leave it here?